crates/sui-indexer/src/apis/read_api.rs

```diff
@@ -78,10 +82,17 @@ impl<S: IndexerStore> ReadApi<S> {
             ),
             // TODO(gegaowp): implement Move call query handling.
             TransactionQuery::MoveFunction {
-                package: _,
-                module: _,
-                function: _,
-            } => Ok(vec![]),
+                package,
+                module,
+                function,
+            } => self.state.get_transaction_digest_page_by_move_call(
```

Comment:
Just wanted to make sure, are de-duplicating the results as multiple rows can share the same txn digest?